jobs/aws/eks-distro-build-tooling/eks-distro-base-periodics.yaml

```diff
@@ -40,7 +40,7 @@ periodics:
       - bash
       - -c
       - >
-        ./eks-distro-base/check_update.sh
+        make update -C eks-distro-base
```

Comment:
Based on our discussions, did it make sense to make this change? Let's wait until we figure out the Makefile target execution in the script situation.

Reply:
Yeah sounds good. I'll still let this be open in case we need to come back to it.